packages/snaps-controllers/src/cronjob/CronjobController.ts

```diff
@@ -266,7 +266,10 @@ export class CronjobController extends BaseController<
     const parsed = parseCronExpression(job.expression);
     const next = parsed.next();
     const now = new Date();
-    const ms = next.getTime() - now.getTime();
+
+    // If the time until the next run is less than 15 seconds, we increase it
+    // to 15 seconds to avoid running jobs too frequently.
+    const ms = Math.max(next.getTime() - now.getTime(), 15_000);
```

Comment:
What if the job is a job that runs every day but meant to start in 1 second?

Reply:
Good point, I need to rethink my approach 😅